### PR TITLE
Update to 1.16.1

### PR DESCRIPTION
**Pull request by MattBDev**
The 1.16.1 update was very insignificant for PlotSquared. Only one change was made and I noticed some missing blocks in an if statement.
I have not tested PlotSquared on 1.16.1 yet even with these changes but I feel it is safe to say that it is working.

### CODE REVIEW

File: Bukkit/src/main/java/com/plotsquared/bukkit/BukkitMain.java [hunk 1]
```diff
@@ -790,7 +791,6 @@ private void startUuidCatching(@NotNull final SQLiteUUIDService sqLiteUUIDServic
                         case MUSHROOM_COW:
                         case OCELOT:
                         case PIG:
```
Comment (Sauilitired) [inline Bukkit/src/main/java/com/plotsquared/bukkit/BukkitMain.java:793]: This needs to be solved differently, possibly by using the names. We need this version to be backward compatible to 1.13. This should also include the new mobs, such as zombified piglins, piglins and hoglins.

Comment (MattBDev) [reply Bukkit/src/main/java/com/plotsquared/bukkit/BukkitMain.java]: Resolved in the new commit.

File: Bukkit/src/main/java/com/plotsquared/bukkit/listener/PlayerEvents.java [hunk 2]
```diff
@@ -1389,6 +1389,11 @@ public void onFade(BlockFadeEvent event) {
             case BUBBLE_CORAL_FAN:
             case FIRE_CORAL_FAN:
             case HORN_CORAL_FAN:
+            case BRAIN_CORAL_WALL_FAN:
+            case BUBBLE_CORAL_WALL_FAN:
+            case FIRE_CORAL_WALL_FAN:
+            case HORN_CORAL_WALL_FAN:
+            case TUBE_CORAL_WALL_FAN:
```
Comment (dordsor21) [inline Bukkit/src/main/java/com/plotsquared/bukkit/listener/PlayerEvents.java:1396]: If we're worried about entity types causing issues, might this also be something we're worried about?

Comment (N0tMyFaultOG) [reply Bukkit/src/main/java/com/plotsquared/bukkit/listener/PlayerEvents.java:1396]: What do you mean? This is not related to 1.16.

Comment (Sauilitired) [reply Bukkit/src/main/java/com/plotsquared/bukkit/listener/PlayerEvents.java:1396]: New ones shouldn't cause issues. The issue is when removing enums, as older server versions won't get the same level of protection

Comment (dordsor21) [reply Bukkit/src/main/java/com/plotsquared/bukkit/listener/PlayerEvents.java:1396]: That makes sense

Comment (MattBDev) [reply Bukkit/src/main/java/com/plotsquared/bukkit/listener/PlayerEvents.java:1396]: These were added to Minecraft awhile ago so it shouldn't cause any compatibility problems.

**Pull request merged**